llvm/test/Transforms/InstCombine/select.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
-; RUN: opt < %s -passes=instcombine -S | FileCheck %s
+; RUN: opt < %s -passes='instcombine<no-verify-fixpoint>' -S | FileCheck %s
```

Comment:
Can you please add inbounds instead...?

Reply:
See https://github.com/llvm/llvm-project/pull/99492#discussion_r1682864397. I cannot reproduce this bug with `inbounds`.

Reply:
See https://github.com/llvm/llvm-project/pull/99492#discussion_r1682885906 :)